packages/css/src/link/index.scss

```diff
@@ -0,0 +1,66 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright (c) 2023 Gemeente Amsterdam
+ */
+
+@import "../../node_modules/@amsterdam/design-system-tokens/dist/variables";
+
+@import "./mixins";
+@import "../../utils/calculate-fluid-style";
+
+.amsterdam-link {
+  @include amsterdam-link;
```

Comment:
I don't think we need these mixins yet, we can add these styles directly in the class

Reply:
Adjusted